### PR TITLE
Add goto url configuration parameter

### PR DESCRIPTION
**Pull request by rghose**
This helps openam redirect back to the application url.

### CODE REVIEW

File: lib/openam_auth/openam.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ class Openam
 
     def initialize
       @base_url = OpenamConfig.openam_url
+			@self_url = OpenamConfig.self_url
```
Comment (morhekil) [inline lib/openam_auth/openam.rb]: could you fix the formatting on this line?

**Comment by morhekil** [comment]
Hi Rahul,

this looks good, thank you for the sending this in. Could you just fix the formatting there, and it will be good to go.

**Comment by rghose** [comment]
Ok done.

**Comment by morhekil** [comment]
Thank you! :+1:

**Pull request merged**